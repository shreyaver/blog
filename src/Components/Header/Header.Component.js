import React from 'react';
import './Header.Component.css';

const Header = (props) => {
    return (
        <header>
        <div className="Content">
            <div>
            <h1>The Artifact</h1>
            <h2>Culture & Art blog</h2>
            </div>
            <nav>
                    <div className="navBar">
                    <a href="/">Blog</a> 
                    <a href="#">About</a>  
                    <a href="#">Contact</a>  
                    </div>
            </nav>
        </div>
        </header>
    )
}
export default Header;
