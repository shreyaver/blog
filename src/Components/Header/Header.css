header {
    background-color: black;
    color: white;
}
.Content {
    width: 70%;
    min-width: 300px;
    margin: auto;
    display: flex;
    justify-content: space-between;
    align-items: flex-end;
    padding-top: 0.5%; 
    padding-bottom: 0.5%; 
}
header h1 {
    margin-bottom: 0px;
    font-size: 20px;
    font-weight: normal;
}
h2 {
    margin-top: 0px;
    font-size: 15px;
    font-style: italic;
    font-weight: normal;
}
nav {
    font-size: 15px;
    width: 40%;
    min-width: 250px;
}
.navBar {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    width: 100%;
}
nav a{
    text-decoration: none;
    color: white;
}
a:hover {
    color: rgb(202, 201, 201);
}