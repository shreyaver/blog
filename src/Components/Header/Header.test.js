import React from 'react';
import renderer from 'react-test-renderer';
import Header from '.';

describe('Header', () => {
  it('renders without crashing', () => {
    const header = renderer.create(<Header />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
