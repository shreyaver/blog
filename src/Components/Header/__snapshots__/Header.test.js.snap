// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<header>
  <div
    className="Content"
  >
    <div>
      <h1>
        The Artifact
      </h1>
      <h2>
        Culture & Art blog
      </h2>
    </div>
    <nav>
      <div
        className="navBar"
      >
        <a
          href="#"
        >
          Blog
        </a>
        <a
          href="#"
        >
          About
        </a>
        <a
          href="#"
        >
          Contact
        </a>
      </div>
    </nav>
  </div>
</header>
`;
